src/TodoList.js

```diff
@@ -0,0 +1,33 @@
+import React, {useState} from "react";
+import TodoItem from "./TodoItem";
+
+const TodoList = () => {
+    let id = 0;
+    const [todos, setTodos] = useState([{
```

Comment:
bringing a non-unique id object can cause bugs
Try using the uuid4 package that generates a random id

Reply:
> I saw that you don't have the package.json file Did you do create-react-app at the beginning of the job?

Yes I needed to run the App somehow :) I wanted to make it more easy to download and update the repository and only uploaded the public and src folders. I would like you to explain us in the next class how git repository looks like and file should be inside

Reply:
> bringing a non-unique id object can cause bugs Try using the uuid4 package that generates a random id

Got it, next time I'll use it!